src/common/darktable.c

```diff
@@ -413,6 +414,14 @@ int dt_init(int argc, char *argv[], const int init_gui, lua_State *L)
   }
 #endif
 
+  if(!dt_openmp_init_stacksize())
+  {
+    fprintf(stderr, "[dt_init] your system uses a low default stacksize and we could not set\n");
+    fprintf(stderr, "[dt_init] the OMP_STACKSIZE environment variable to %dB.\n", SAFESTACKSIZE);
+    fprintf(stderr, "[dt_init] Please set the OMP_STACKSIZE environment variable before starting darktable\n");
+    return 1;
+  }
+
```

Comment:
If you're going to make dt complain about a low value here, then you should return the detected value from dt_openmp_init_stacksize() so that you can provide the user with information about how far off the minimum they are. 

is SAFESTACKSIZE a minimum or a maximum? This information should also be provided to the user.